docs/src/app/[locale]/(home)/tools/components/ToolPageLayout.module.css

```
@@ -0,0 +1,148 @@
+.workspace {
+  min-width: 0;
+  width: 100%;
+  color: var(--foreground);
+}
+
+.workspace pre {
+  max-width: 100%;
+  overflow-x: auto;
+}
+
+.workspace > :global(.mb-16),
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)